src/HeaderBag.php

```diff
@@ -103,6 +106,6 @@ public function getValue()
 
     public function __toString()
     {
-        return $this->name . ($this->value === '' ? '' : ': ' . $this->value);
+        return $this->name . ':' .(empty($this->value) ? '' : ' ' . $this->value);
```

Comment:
Both bugs fixed here were brought over from my code of course, just explaining the change here ;-)